client/export.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 	"io"
 	"net/http"
+	"net/http/httptrace"
```

Comment:
Unused import

Reply:
Sorry, expected the commit hook in git to find all those.

Fixed.

Reply:
The git hook we typically use only works in our closed-source repo (since it refers to scripts in that repo.) You'd have to manually setup a commit hook for these repos or configure your own global hook in `$HOME/.gitconfig`.

Reply:
There's also a commit hook you can add for this repo that I added with #56